zppy/defaults/default.ini

```diff
@@ -107,9 +107,11 @@ input_component = string(default="")
 [ts]
 area_nm = string(default="area")
 cmip_metadata = string(default="inclusions/e3sm_to_cmip/default_metadata.json")
+cmip_plevdata = string(default="inclusions/e3sm_to_cmip/vrt_remap_plev19.nc")
```

Comment:
```bash
du -sh vrt_remap_plev19.nc 
# 512	vrt_remap_plev19.nc
du -sh default_metadata.json 
# 8.0K	default_metadata.json
```
I can't tell the units of the `du` measurement here; I think it's just bytes. How big is this file? Less than a kilobyte? I guess that's ok, but we typically avoid including `nc` files in the repo.

Reply:
This is pretty small, so should be ok to include in the repo -- or we could add this file to the mapping directory, for consistency.